umap/static/umap/js/umap.controls.js

```diff
@@ -650,6 +650,40 @@ const ControlsMixin = {
         this.options.user.url
       )
     }
+
+    const connectedPeers = this.sync.getNumberOfConnectedPeers()
+    if (connectedPeers !== 0) {
+      const connectedPeersCount = L.DomUtil.createButton(
+        'leaflet-control-connected-peers',
+        rightContainer,
+        '',
+        this.displayConnectedPeers,
```

Comment:
I don't get where is this method defined, can you point me to it , :)

Reply:
It's currently not implemented, I removed it for now, it can come later on.